plugins/obs-ffmpeg/obs-ffmpeg-vaapi.c

```diff
@@ -44,6 +45,9 @@
 #include "vaapi-utils.h"
 #include "obs-ffmpeg-formats.h"
 
+#include <va/va_drmcommon.h>
+#include <libdrm/drm_fourcc.h>
```

Comment:
Libdrm needs to be added in CMake as a dependency

Reply:
Only linux-pipewire uses libdrm.